docs/using_python_client_with_hazelcast_imdg.rst

```diff
@@ -2137,6 +2137,50 @@ See the following example.
     # Average age is 30
     print("Average age is %f" % average_age)
 
+Projections
+~~~~~~~~~~~~~~~~~
+
+Projections feature provides three projection functions; ``single_attribute``, 
+``multi_attribute`` and ``identity``, on top of Hazelcast ``Map`` entries. Their
+performance is high since they run in parallel for each partition and are 
+highly optimized for speed and low memory consumption.
+
+These projections are used with the ``map.project`` function, which takes an
+optional predicate argument.
+
+See the following example.
```

Comment:
```suggestion
There are cases where instead of sending all the data returned by a query
from the server, you want to transform (strip down) each result object in order
to avoid redundant network traffic.

For example, you select all employees based on some criteria, but you just
want to return their name instead of the whole object. It is easily doable
with the Projections.

The ``projection`` module provides three projection functions:

- ``single_attribute``: Extracts a single attribute from an object and returns
  it.
- ``multi_attribute``: Extracts multiple attributes from an object and returns
  them as a ``list``.
- ``identity``: Returns the object as it is.

These projections are used with the ``map.project`` function, which takes an
optional predicate argument.

See the following example.
```
Let's replace the docs with this.